src/cc_helpers.cc

```diff
@@ -427,6 +437,29 @@ bool certifier::framework::cc_trust_data::put_trust_data_in_store() {
     return false;
   }
 
+  const string pka("public_key_algorithm");
+  int pki = store_.get_blob_index_by_tag(pka);
+  if (pki >= 0) {
+    tagged_blob_message* tbp = store_.tagged_blob_[pki];
+    tbp->set_b(public_key_algorithm_.data(), public_key_algorithm_.size()+1);
+  } else {
+    if (!store_.add_blob(pka, public_key_algorithm_)) {
+      printf("cold_init: Can't add public key algorithm to store\n");
```

Comment:
Ins't `cold_init` in the message misleading here?

I see there is already another method by the same name:

```
 633 bool certifier::framework::cc_trust_data::cold_init(const string& public_key_alg,
```

But this error is coming from `cc_trust_data::put_trust_data_in_store`.

Please try to see if `__func__` reports the desired method's name.

Same comment applies to L458 below.